### PR TITLE
Create a class for admin form

### LINKED ISSUE #111: Refactor out the form-generation from Admin.php to Form.php
Admin.php is too heavy and is doing too much. As discussed, let's separate concerns regarding template generation.

- Move the template-generating PHP into its own file
- Define the fields in a class property (share debug data, create support ticket, ticket description)
- Perhaps create a method like `Form::get_submitted_fields()` that can be used by the Ajax class so that it can be used to verify requests instead of [defining inline the fields that exist](https://github.com/trustedlogin/client/blob/c1da3d556a8dfe8b44e7bdcf70d1073b84c191bb/src/Ajax.php#L88)

### PR DESCRIPTION
**Pull request by Shelob9**
Resolves #111

### CODE REVIEW

**Comment by Shelob9** [comment]
@zackkatz I tested this, with wp-env locally using same changes as #115.

It works to grant access. JS/CSS is loaded. Not sure if this is compilation error or sign of a problem:
Screenshot 2023-04-05 135432

I ran `grunt` got this error on SAAS rebuild `You need to have Ruby and Sass installed and in your PATH for this task to work.` I will try to solve for that locally, but this is ready for review.

File: src/Form.php [hunk 1]
```diff
@@ -0,0 +1,1202 @@
...
+
+		$footer_output = '';
+		if (!empty($footer_links_output)) {
+			$footer_output = sprintf('<ul>%1$s</ul>', $footer_links_output);
+		}
+
+		return $footer_output;
+	}
+
+	private function prepare_output($template, $content, $wp_kses = true)
```
Comment (zackkatz) [inline src/Form.php:648]: I feel like this should be a static method in its own `Helpers.php` file or something like that.

File: src/Form.php [hunk 2]
```diff
@@ -0,0 +1,1202 @@
...
+
+		$displayed_notice = true;
+	}
+
+	/**
+	 * Is this a login screen and should TrustedLogin override the login screen for the current namespace?
+	 *
+	 * @return bool
+	 */
+	private function is_login_screen()
```
Comment (zackkatz) [inline src/Form.php:1198]: Move back to Admin?

File: src/Form.php [hunk 3]
```diff
@@ -0,0 +1,1202 @@
...
+	 * @var null|Logging $logging
+	 */
+	private $logging;
+
+	/**
+	 * Admin constructor.
+	 *
+	 * @param Config $config
+	 */
+	public function __construct(Config $config, Logging $logging,SupportUser $support_user, SiteAccess $site_access)
```
Comment (zackkatz) [inline src/Form.php:64]: Formatting isn't consistent with other files; I can fix this though.

**Review by zackkatz — APPROVED** [review]
Looks good. Check out my few comments.

File: src/Form.php (hunk 2 above)
Comment (Shelob9) [reply src/Form.php:1198]: Its not used in Admin.

File: src/Form.php (hunk 1 above)
Comment (Shelob9) [reply src/Form.php:648]: It's only need  in this class though

File: src/Form.php (hunk 3 above)
Comment (Shelob9) [reply src/Form.php]: 9d5b85614e674bb6c6e2d135beea81a93ff45f0b

**Comment by zackkatz** [comment]
Already merged in #116

**Pull request closed without merging**